lib/nerves_network/dhcp_manager.ex

```diff
@@ -389,8 +328,68 @@ defmodule Nerves.Network.DHCPManager do
 
   # Catch-all handler for consume
   defp consume(context, event, state) do
-    Logger.warn("Unhandled event #{inspect(event)} for context #{inspect(context)} in consume/3.")
+    Logger.warning("Unhandled event #{inspect(event)} for context #{inspect(context)} in consume/3.")
+    state
+  end
+
+  # When the netmask is empty (no such entry in the leases DB fetch the netmask information directly from
```

Comment:
it looks like this section hasn't changed. is that right? probably GH just doing a poor comparison, as below code looks the same to me as the previous but higher up instead.

Reply:
It is mostly the same yes, just getting rid of warnings as they were present/arose.